api/messaging/spec/webhooks.json

```diff
@@ -0,0 +1,107 @@
+{
+  "Title": "Webhooks",
+  "Interface": "Webhooks",
+  "Struct": null,
+  "Parameters": {},
+  "Protocol": "",
+  "Authentication": [],
+  "Path": "/webhooks",
+  "APIs": [
+    {
+      "Name": "webhookGet",
+      "Method": "GET",
+      "Title": "Get webhook details",
+      "Path": "/webhook/{webhookID}",
+      "Parameters": {
+        "path": [
+          {
+            "name": "webhookID",
+            "required": true,
+            "title": "Webhook ID",
+            "type": "uint64"
+          }
+        ]
+      }
+    },
+    {
+      "Name": "webhookDelete",
+      "Method": "DELETE",
+      "Title": "Delete webhook",
+      "Path": "/webhook/{webhookID}",
+      "Parameters": {
+        "path": [
+          {
+            "name": "webhookID",
+            "required": true,
+            "title": "Webhook ID",
+            "type": "uint64"
+          }
+        ]
+      }
+    },
+    {
+      "Name": "webhookDeletePublic",
+      "Method": "DELETE",
+      "Title": "Delete webhook (public)",
+      "Path": "/webhook/{webhookID}/{webhookToken}",
+      "Parameters": {
+        "path": [
+          {
+            "name": "webhookID",
+            "required": true,
+            "title": "Webhook ID",
+            "type": "uint64"
+          },
+          {
+            "name": "webhookToken",
```

Comment:
should this be at least GET if not header?

Reply:
I suggest we (re)use JWT here, no need for additional authentication here.

Reply:
Agreed. JWT should be used on API routes for validation. Effectively it's going to be a 100 year expiry, so we'll still need to validate it against the DB. Will resolve comment thread when done.

Reply:
I opted against this, since:

1. the api route doesn't need middleware validation (exclusive route),
2. we don't want to leak the jwt identity claims for a webhook to other APIs,
3. that leaves the only jwt feature that would be used: expiry (which we don't need)

Effectively any random string/uuid will do.